### PR TITLE
Improve developer documentation

### PR DESCRIPTION
**Pull request by marccarre**
These are the steps I personally followed. There may be "better" ones, but these worked.
I'm sharing these in the form of this PR as this could make things easier for other contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,3 +103,54 @@ None
...
+    $ cd /path/to/tinycards-python-api
+    $ virtualenv .
+    $ source bin/activate
+    (tinycards-python-api) $
+    ```
+
+- Install dependencies within the virtual environment:
+
+    ```console
+    (tinycards-python-api) $ pip install -r requirements.txt
```
[the remark this answers is not in the document]
Comment (marccarre) [reply README.md]: Fixed 👍

File: README.md [hunk 2]
```diff
@@ -112,3 +103,54 @@ None
...
+    (tinycards-python-api) $ pip install -r test-requirements.txt
+    ```
+
+- Develop and test at will.
```
[the remark this answers is not in the document]
Comment (marccarre) [reply README.md]: Fixed 👍

File: README.md [hunk 3]
```diff
@@ -112,3 +103,54 @@ None
...
+    $ echo "export TINYCARDS_PASSWORD=<pass>" >> .envrc
+    $ direnv allow
+    direnv: loading .envrc
+    direnv: export +TINYCARDS_IDENTIFIER +TINYCARDS_PASSWORD
+    ```
+
+2. Then, from the project's root directory, simply start the `pytest` test runner:
+
+    ```console
+    $ pytest tinycards
```
[the remark this answers is not in the document]
Comment (marccarre) [reply README.md]: Fixed 👍

**Comment by floscha** [comment]
Looks good to me now, so I'll merge the PR. Thanks again for your effort!

**Pull request merged**